### ISSUE TITLE
Non-stacked bar series on separate value axes

### ISSUE THREAD

**Issue by adamncsu** [oxyplot/oxyplot#1777]
I'm trying to put two bar series on separate value axes. 

Here's the plot with both axes displayed, but both series are using the first axis ("x1"):
![Capture1](https://user-images.githubusercontent.com/642646/128560219-be003b1f-83c9-40b7-b9c6-04068a11d344.PNG)

So I change the second series to use the second value axis ("x2"), and they become overlapped:
![Capture2](https://user-images.githubusercontent.com/642646/128560277-23a5dda0-804e-4c44-8015-07a824ccc088.PNG)

Here's my code:
```
model = new PlotModel();

var series1 = new BarSeries() { XAxisKey = "x1" };
series1.Items.Add(new BarItem { Value = 1 });
series1.Items.Add(new BarItem { Value = 2 });
model.Series.Add(series1);

var series2 = new BarSeries() { XAxisKey = "x2" };
series2.Items.Add(new BarItem { Value = 3 });
series2.Items.Add(new BarItem { Value = 1 });
model.Series.Add(series2);

var categoryAxis = new CategoryAxis { Position = AxisPosition.Left };
model.Axes.Add(categoryAxis);

var valueAxis1 = new LinearAxis { Key = "x1", Position = AxisPosition.Bottom, Minimum = 0, Maximum = 3 };
var valueAxis2 = new LinearAxis { Key = "x2", Position = AxisPosition.Bottom, Minimum = 0, Maximum = 3, PositionTier = 2 };
model.Axes.Add(valueAxis1);
model.Axes.Add(valueAxis2);
```

**Comment by VisualMelon** [comment]
The `BarSeriesManager` currently groups series by the category axis and value axis, so this isn't supported.

You can probably work around this by extending `BarSeries` and implementing `IBarSeries.ValueAxis` explicitly so that it returns a common value (e.g. the `CategoryAxis`); that will trick the bar series grouping them together.

As a proper solution, we could allow bar series to supply their own group name (as they can provide their own stack groups).

Pinging @Jonarw who re-wrote all this recently.

**Comment by mandreou** [comment]
Hello,

i'm working on something with barseries and category axis. I have a list with two doubles in each node. First double represents the category axis and second value the barserie.

Imagine the list is : {1,5}{2,4}{2,7}{3,4.5}{5,5}{5.4,5}{5.4, 6}

Category axis has the numbers 1, 2, 3, 5, 5.4, 
Baseries has 5, 4, 7, 4.5, 5, 5.4

So on the same category axis with value 5.4 i want to have two bars with value 5 and 6 respectively
So basically overlapping. is that possible please?

**Comment by VisualMelon** [comment]
@mandreou I'm not entirely clear what you mean, but the `IsStacked` and `OverlapsStacked` properties of `BarSeries` may help.

**Comment by mandreou** [comment]
![Capture](https://user-images.githubusercontent.com/31207126/160821461-60085e82-528f-461b-bfbe-388547b7537f.PNG)


something like that. 
Imagine you have a tuple <double,double> {1,5}{2,4}{2,7}{3,4.5}{5,5}{5.4,5}{5.4, 6}

if left double number exists twice it means that the right double number will be overlapping it.
in the position 5.4 of the category axis we have 5 & 6 values for the barseries. Each bar will have different colour if belong in same category axis value
